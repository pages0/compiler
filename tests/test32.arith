let x : <int> = ref 1 in !x