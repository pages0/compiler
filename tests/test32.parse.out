( let [ x ( ref 1 ) ] ( ! x ) )